openwisp_users/apps.py

```diff
@@ -130,6 +137,17 @@ def connect_receivers(self):
             dispatch_uid='make_first_org_user_org_owner',
         )
 
+    @classmethod
+    def handle_organization_update(cls, instance, **kwargs):
+        try:
+            old_instance = instance._meta.model.objects.get(pk=instance.pk)
```

Comment:
```suggestion
            old_instance = Organization.objects.only('is_active').get(pk=instance.pk)
```

Calling `only` should be more efficient.